.dir-locals.el

```diff
@@ -0,0 +1,4 @@
+((nil . ((indent-tabs-mode . t)
```

Comment:
Also, I'm fairly certain most of us use tab=4sp, which would account for some of the weird shifts.